Formula/direvent.rb

```diff
@@ -0,0 +1,19 @@
+class Direvent < Formula
+  desc "Monitors events in the file system directories"
+  homepage "http://www.gnu.org.ua/software/direvent/direvent.html"
+  url "https://ftpmirror.gnu.org/direvent/direvent-5.1.tar.gz"
+  sha256 "c461600d24183563a4ea47c2fd806037a43354ea68014646b424ac797a959bdb"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make"
+    system "make", "-C", "tests", "check-local"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/direvent", "--version"
```

Comment:
Let's assert on the version here. It's not much stronger, but it's something:
```ruby
assert_match version.to_s, shell_output("#{bin}/direvent")
```

Even better though if you could think up a functionality test? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test.